planet/clients/subscriptions.py

```diff
@@ -22,15 +25,23 @@ class SubscriptionsClient:
     exceptions raised by this class are categorized by HTTP client
     (4xx) or server (5xx) errors. This client's level of abstraction is
     low.
-
-    Attributes:
-        session (Session): an authenticated session which wraps the
-            low-level HTTP client.
     """
 
-    def __init__(self, session: Session) -> None:
+    def __init__(self,
+                 session: Session,
+                 base_url: Optional[str] = None) -> None:
+        """
+        Parameters:
+            session: Open session connected to server.
+            base_url: The base URL to use. Defaults to production subscriptions
```

Comment:
I'm not sure I understand this client well enough, but what's the purpose of defining a base URL aside from the subscriptions API base URL?

Reply:
Redirecting to a test service is the primary use case. @cholmes has been using it for that. Also, QA team could redirect to a QA service.